### PR TITLE
Correct `<Exec />` commands on Windows to handle spaces in root path

### PR DESCRIPTION
**Pull request by dougbu**
- #15055
- add a few more quotation marks

### CODE REVIEW

**Review by BrennanConroy — APPROVED** [review]
No complaints

File: src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj [hunk 1]
```diff
@@ -209,8 +209,8 @@ This package is an internal implementation of the .NET Core SDK and is not meant
       Overwrite="true" />
 
     <!-- Requires Windows 10 version 1803 or newer -->
-    <Message Importance="High" Text="Executing: $(_TarCommand) -czf $(_TarArchiveOutputPath) ." />
-    <Exec Command="$(_TarCommand) -czf $(_TarArchiveOutputPath) ."
+    <Message Importance="High" Text="Executing: $(_TarCommand) -czf &quot;$(_TarArchiveOutputPath)&quot; ." />
+    <Exec Command="$(_TarCommand) -czf &quot;$(_TarArchiveOutputPath)&quot; ."
```
Comment (pranavkm) [inline src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj]: Does the command need quotes?

**Review by pranavkm — APPROVED** [review]

**Review by wtgodbe — APPROVED** [review]

File: src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj (hunk 1 above)
Comment (dougbu) [reply src/Framework/ref/Microsoft.AspNetCore.App.Ref.csproj]: Slight corner case but, nonetheless, I'm double-checking this one as well as a few more of our eng/scripts scripts&hellip;

**Pull request merged**